src/components/TextInput/TextInput.tsx

```diff
@@ -17,14 +17,18 @@ export function TextInput ( props: {
 	autoComplete:
 		'country' |
 		'email' |
+		'family-name' |
+		'given-name' |
 		'name' |
+		'new-password' |
 		'off' |
 		'on' |
 		'organization' |
+		'password' |
 		'street-address' |
 		'tel' |
-		'username' |
-		'url',
+		'url' |
+		'username',
```

Comment:
> Is there a type for this I can just import?

I had to Google this but apparently yes! [This is possible](https://stackoverflow.com/questions/36311284/is-there-a-way-to-extract-the-type-of-typescript-interface-property) using "indexed access types" aka "lookup types".

Judging by the TS tooltip for `autoComplete`, the full type is `React.InputHTMLAttributes<HTMLInputElement>.autoComplete`:

<img width="573" alt="Screen Shot 2021-04-07 at 9 38 06 AM" src="https://user-images.githubusercontent.com/784999/113875930-3c1e5f00-9785-11eb-8178-5af23666a744.png">


So using a lookup type you could change this to simply:

```ts
autoComplete?: React.InputHTMLAttributes<HTMLInputElement>['autoComplete'],
```

Reply:
I think the linter will complain about this, because I think the linter is configured incorrectly. You'll also need to change the parser in `.eslintrc` to:

```json
"parser": "@typescript-eslint/parser",
```

Reply:
omg @hallee you're a rock star. Hey, if it's a string, then it's a string